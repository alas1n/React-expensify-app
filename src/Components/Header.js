import React from 'react'
import {NavLink } from 'react-router-dom'



const Header = () => (
    <div>
        <h1> Expense App </h1>
        <NavLink to="/" activeClassName="is-active" exact> home </NavLink> <br></br>
        <NavLink to="/create" activeClassName="is-active"> create </NavLink> <br></br>
        <NavLink to="/edit" activeClassName="is-active"> edit </NavLink> <br></br>
        <NavLink to="/help" activeClassName="is-active"> help </NavLink> <br></br>
    </div>
)

export default Header