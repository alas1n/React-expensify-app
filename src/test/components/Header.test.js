import React from 'react'
import {shallow} from 'enzyme'
import Header from '../../Components/Header'


test('should render Header correctly', () => {
	const wrapper = shallow(<Header />)
	expect(wrapper).toMatchSnapshot()
})
