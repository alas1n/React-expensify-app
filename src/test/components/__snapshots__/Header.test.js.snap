// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
     Expense App 
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
     home 
  </NavLink>
   
  <br />
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
     create 
  </NavLink>
   
  <br />
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/edit"
  >
     edit 
  </NavLink>
   
  <br />
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
     help 
  </NavLink>
   
  <br />
</div>
`;
